eng/helix.proj

```diff
@@ -88,6 +89,10 @@
       <Destination>test\BaselineOutput</Destination>
     </HelixCorrelationPayload>
   </ItemGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Templating" Version="$(MicrosoftDotNetBuildTasksTemplatingVersion)" PrivateAssets="All" />
```

Comment:
Looks like the current pipeline doesn't restore this project https://github.com/dotnet/machinelearning/blob/01c41644edef3d02060b4c8bb841878df3528050/build/ci/send-to-helix.yml#L24